.github/workflows/gradle.yml

```diff
@@ -25,5 +25,11 @@ jobs:
           ${{ runner.os }}-gradle-
     - name: Build with Gradle
       run: ./gradlew build
+    - name: Archive test results
+      uses: actions/upload-artifact@v2
```

Comment:
Might be good to be able to download the test artifacts.
https://docs.github.com/en/free-pro-team@latest/actions/guides/storing-workflow-data-as-artifacts